modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -63,6 +65,22 @@ imageContentSourcePolicy.yaml
 mapping.txt
 ----
 
+. Create a CatalogSource object for the new catalog:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: my-operator-catalog
+  namespace: openshift-marketplace
+spec:
+  sourceType: grpc
+  image: <registry_host_name>:<port>/<namespace>/redhat-operators:v1
+  displayName: My Operator Catalog
+  publisher: grpc
+----
+
 . Apply the manifests:
 +
 ----
```

Comment:
I'd suggest that adding a step to create this `CatalogSource` object before step 4. Such as: `oc create -f xxx.yaml(the YAML name)`